### PR TITLE
Fix editorconfig violations

### PR DESCRIPTION
**Pull request by szepeviktor**
@TomasVotruba 🚀

### CODE REVIEW

File: config/template-compiler/config.neon [hunk 1]
```diff
@@ -1,3 +1,3 @@
 conditionalTags:
-	PhpParser\NodeVisitor\NodeConnectingVisitor:
-		phpstan.parser.richParserNodeVisitor: true
+    PhpParser\NodeVisitor\NodeConnectingVisitor:
+        phpstan.parser.richParserNodeVisitor: true
```
Comment (TomasVotruba) [inline config/template-compiler/config.neon:3]: :+1:

**Comment by TomasVotruba** [comment]
This look good :+1: , just breaks one tests. 

Could you update it and maybe ignore it on test fixtures? They'll be generated by rought tool, so no need to spent time on spacing there.

**Comment by szepeviktor** [comment]
Working on it ...
🏃🏻‍♂️

**Comment by TomasVotruba** [comment]
Thanks :clap:

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**